shaders/clouds.omwfx

```diff
@@ -513,9 +502,25 @@ fragment main {
             * max(sun_dir.z, 0.0) * 3.0
             // Regulate direct lighting with max_dist to prevent it appearing on top of close objects
             * min(max_dist * (1.0 / 100000.0), 1.0);
-        vec3 fog_color = (sun_light + moon_light) * mix(sky_color, sky_tone, max(sun_dir.z, 0.3));
-
-        vec3 fog_light = fog_color + direct_light * sun_color;
+        //Define the atmospheric attenuation factors for sunlight's red, green, and blue channels
+        vec3 att_factor = vec3(
+            (pow((2 / (abs(sun_dir.z) + 1)), 2) / 2),
+            (sqrt(1 / (abs(sun_dir.z))) / 10 * (abs(sun_dir.z))),
+            (pow((1 / (abs(sun_dir.z) + 1)), 2) * 3 * (-1 * (abs(sun_dir.z))))
```

Comment:
Make sure to use float notation (i.e: `1.0` and `2.0` instead of `2` and `1`), a lot of drivers get angry if you try to use integers where floats are expected.